roles/github/templates/run-playbook.yml.j2

```diff
@@ -0,0 +1,67 @@
+<%- if github_environment_selector == 'input' -%>
```

Comment:
There are some differences between this and generic.yml.j2 and it's not clear why:
- No concurrency group
- No timeout
- Slight difference in checkout
- Symlink vs copy kayobe config
- Run pipeline script using sudo -E -u stack bash

Reply:
Given the scope for inconsistency here, and the small necessary differences between this and generic.yml.j2, would it be better to allow setting the pipeline script and providing CLI arguments to it using the existing generic.yml.j2 template?

Reply:
Yeah that seems like a sensible approach. Will update the PR once changes have been made.